src/mmw/js/src/core/templates/layerControlContainer.html

```diff
@@ -0,0 +1,23 @@
+<div>
+    <div class="layer-control-button-container leaflet-control leaflet-bar">
+        <a class="leaflet-bar-part leaflet-bar-part-single" href="#" title="Show layer control">
+            <span class="fa fa-globe"></span>
```

Comment:
Annoying. I generally think we should be using fontello (http://fontello.com/) to pick icons we need so we don't have to load 500 others that never get used.